orc8r/cloud/go/orc8r/math/math.go

```diff
@@ -28,3 +28,17 @@ func JitterUint32(n uint32, key string, maxMultiplier float32) uint32 {
 	maxJitter := float32(n) * maxMultiplier
 	return uint32(multiplier * maxJitter)
 }
+
+// JitterInt64 returns a jitter of the given int64 value that is deterministic
+// based on the given key.
+func JitterInt64(n int64, key string, maxMultiplier float64) int64 {
+	// FNV-1 is a non-cryptographic hash function that is fast and very simple to implement
```

Comment:
If we do keep this JitterInt64 around, pull out the part where we generate a deterministic int from the key